services/api-server/src/simcore_service_api_server/exceptions/handlers/_utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Awaitable, Callable, TypeAlias
+
+from fastapi.encoders import jsonable_encoder
+from fastapi.requests import Request
+from fastapi.responses import JSONResponse
+
+from ...models.schemas.errors import ErrorGet
+
+ExceptionHandler: TypeAlias = Callable[
+    [Request, BaseException], Awaitable[JSONResponse]
+]
+
+
+def create_error_json_response(
+    *errors: Any, status_code: int, **kwargs
+) -> JSONResponse:
+    """
+    Converts errors to Error response model defined in the OAS
+    """
+
+    # NOTE: do not forget to add in the decorator `responses={ ???: {"model": ErrorGet} }`
```

Comment:
Q: is this a todo or referred to the user of this function? it's not clear to me

Reply:
thx. it is an old comment